src/test/authentication/middleware/index.spec.unit.ts

```diff
@@ -41,6 +41,13 @@ describe("Authentication middleware", () => {
     expect(response.status).toEqual(302);
   });
 
+  it("should not redirect to signin if loading accessibility statement page", async () => {
+    const response = await request(app)
+      .get("/extensions/accessibility-statement")
+      .set("Referer", "/extensions");
+    expect(response.status).toEqual(200);
```

Comment:
need to check what page is returned, could be returning 200 but for a different page

Reply:
Added a check for the page title